### DISCUSSION TITLE
Experimental bake

### DISCUSSION THREAD

**Discussion by gabs79** [docker/buildx#1833] [Q&A]
Why is Bake still experimental? I love the plugin and command. Any timeframe for non- experimental?
Based on note in: https://docs.docker.com/build/bake/
@tonistiigi any insights?

**Comment by colinhemmings** [comment]
I will cross-post my response from a related issue.

We're glad you are loving Bake. Apologies for taking so long, but don't worry, you won't have to wait much longer. We are fully committed to Bake, and there will be an announcement related to this in January. The team is hard at work on some additional features for Bake to make it really Shine.

The team and I would love to hear which Bake features benefit you the most, and if there are any additional things you would like to see added?
